batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/service/EmployeeService.java

```diff
@@ -16,8 +19,15 @@ public EmployeeService() {
     employeeDao = new EmployeeDao();
   }
 
-  public void insert(Employee employee) throws DataException {
-    employeeDao.insert(employee);
+  public void insert(Employee employee) throws DataException, ValidationException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)